### ISSUE TITLE
SMTP over SSL is not working

### ISSUE THREAD

**Issue by tareksamni** [brotandgames/ciao#17]
I tried to run Ciao and integrate it with Sendgrid. It did work perfectly but I got Timeout errors when the ActiveMailer tried to communicate with Sendgrid SMTP server.

After some investigations, I found that I need to set `ssl: true` in `action_mailer.smtp_settings` to support SMTP over SSL which is required by Sendgrid and probably any other reliable and save SMTP server.

**Comment by brotandgames** [comment]
- Registered an account with sendgrid.net
- Followed the https://sendgrid.com/docs/for-developers/sending-email/rubyonrails/ guide to setup ActionMailer
````
SMTP_ADDRESS:               smtp.sendgrid.net
SMTP_AUTHENTICATION:        plain
SMTP_DOMAIN:                brotandgames.com
SMTP_EMAIL_FROM:            email_from@gmail.com
SMTP_EMAIL_TO:              email_to@gmail.com
SMTP_ENABLE_STARTTLS_AUTO:  auto
SMTP_PASSWORD:              *****
SMTP_PORT:                  587
SMTP_USERNAME:              brotandgames
````
- Created a new Check
<img width="1178" alt="grafik" src="https://user-images.githubusercontent.com/43862266/61986361-0405d580-b00f-11e9-8c97-708f5f83ee17.png">

- E-Mail delivered as wanted

<img width="506" alt="grafik" src="https://user-images.githubusercontent.com/43862266/61986529-04eb3700-b010-11e9-9daa-fdea539a26c7.png">


It seems that you don't need to set `ssl: true` in `action_mailer.smtp_settings`.

**Comment by brotandgames** [comment]
Could you provide you configuration and logs from the application when ActionMailer tries to send an e-mail via Sendgrid?

**Comment by tareksamni** [comment]
‘SMTP port: 25 or 587 for unencrypted/TLS email, 465 for SSL-encrypted email’

Try with port 587. It won’t work :)

**Comment by brotandgames** [comment]
@tareksamni Since Sendgrid supports `STARTTLS` on Port 587 the connection is encrypted/secure.
Nevertheless we will have to support the use of SSL/TLS explicitly on port 465 (also for other SMTP servers/providers).

**Comment by brotandgames** [comment]
Closed with #16
